packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -106,6 +106,11 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   setInitialFocus?: boolean;
 
   /**
+   * Set the max height of Callout component.
```

Comment:
default is null, if nothing passed in here, the logic will use the existing logic:
 let contentMaxHeight: number = this.props.maxHeight || this._getMaxHeight();